dna-frontend/src/app/core/components/barchart/barchart.component.html

```diff
@@ -0,0 +1,7 @@
+<tui-axes
```

Comment:
This applies to all the chart components:
We probably shouldn't wrap them in additional components due to the encapsulation.
Due to the nature of how the axes and charts work, the Taiga UI component for graphs and charts should be placed directly into their parent components instead. This was useful to understand how these charts worked though.

Reply:
- [ ] Remove all chart components in favor of using them directly in parent components